raudio_bot/__main__.py

```diff
@@ -10,6 +10,36 @@
 from dotenv import load_dotenv
 
 
+import json
+
+def check_for_file(file_name):
+    if os.path.exists(file_name):
+        return True
+    else:
+        x = input("Currently, there is no configuration specified for this bot. Would you like to create one? (y/n): ")
+
+        if (x == 'y'):
+            # get url and create data dictionary containing default specifications
+            url = input("Enter URL of server you will be streaming from: ")
+
+            # ADD ANY OTHER DEFAULT CONFIGS HERE
+            data = {'stream_url': url, 'authenticated': [ ]}
+
+            # create default json file in upper directory
+            with open('raudio_config.json','w') as jsonfile:
+                json.dump(data, jsonfile)
+
+            # call recursively on newly created file
+            raudio_config_from_json('raudio_config.json')
```

Comment:
Calling this function again doesn't seem necessary.

```suggestion
```

Reply:
Ya that makes sense. Would I just commit your suggestion to clear this up?